wordle-web/components/SettingsDialog.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-dialog v-model="modelValue" max-width="500">
+    <v-card>
+      <v-sheet color="primary">
+        <v-card-text> Settings </v-card-text>
+      </v-sheet>
+      <v-card-text>
+        Dark/Light:   
+        <v-btn 
+          icon="mdi-theme-light-dark" @click="toggleTheme" />
+      </v-card-text>
+      
+      <v-sheet class="ml-6">
+        <v-select
+          color="secondary"
+          v-model="chosenTheme"
+          label="Choose a Theme"
+          :items="['Standard', 'Mystic Meadow', 'Enchanted Eclipse']"
+        ></v-select>
+        <v-btn color="secondary" @click="changeTheme" class="mb-6" type="submit">Activate</v-btn>
+      </v-sheet>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script setup lang="ts">
+import { useTheme } from "vuetify";
+import nuxtStorage from "nuxt-storage";
+
+const theme = useTheme();
+const modelValue = defineModel<boolean>({ default: false });
+const chosenTheme = ref('Standard');
+
+
+function changeTheme() {
+  const darkMode: boolean = isDark();
+  if(chosenTheme.value === "Mystic Meadow"){
+    if(darkMode){
+      theme.global.name.value = "mysticMeadowDark";
+    }
+    else {
+      theme.global.name.value = "mysticMeadowLight";
+    }
+  }
+  if(chosenTheme.value === "Enchanted Eclipse"){
+    if(darkMode){
+      theme.global.name.value = "enchantedEclipseDark";
+    }
+    else {
+      theme.global.name.value = "enchantedEclipseLight";
+    }
+  }
+  if(chosenTheme.value === "Standard"){
+    if(darkMode){
+      theme.global.name.value = "dark";
+    }
+    else {
+      theme.global.name.value = "light"
+    }
+  }
```

Comment:
```suggestion
  switch (chosenTheme.value) {
    case "Mystic Meadow":
      theme.global.name.value = "mysticMeadow" + (isDark() ? "Dark" : "Light");
      break;
    case "Enchanted Eclipse":
      theme.global.name.value =
        "enchantedEclipse" + (isDark() ? "Dark" : "Light");
      break;
    default:
      theme.global.name.value = isDark() ? "dark" : "light";
      break;
  }
```
Nested if else statement can be simplified using switch statement. There also isn't a need to do isDark bool when you can just call isDark. To make it logic easier to do, I would suggest using a component that you could track the state if it is light or dark mode. `v-switch` could work. Then watch that value and if the value changes, update the theme accordingly to its light or dark varient.

Reply:
Great suggestion on the switch statement, which I did implement.  I'll work with the v-switch to try to get that implemented, I had problems trying to implement it initially.